app/models/bulkrax/csv_entry.rb

```diff
@@ -41,6 +41,7 @@ def build_metadata
       self.parsed_metadata = {}
       add_identifier
       add_ingested_metadata
+      add_collections
```

Comment:
@sephirothkod I don't think this is a good choice for bulkrax because this line seems to have been removed intentionally for some reason. however, without it `self.collection_ids` is always empty so `collections_created?` will return false and throw a CollectionCreatedError. even when the collection is in fact created.

Reply:
@sephirothkod @bkiahstroud actually...I now needed to bring this code into LV and BL through overrides. may it be better to put this code into bulkrax proper, but with a note to reevaluate removing `add_collections` from here in the future once the updated logic is worked out?

Reply:
@sephirothkod @bkiahstroud actually...I now needed to bring this code into LV and BL through overrides. may it be better to put this code into bulkrax proper, but with a note to reevaluate removing `add_collections` from here in the future once the updated logic is worked out?

Reply:
sure lets leave it for now and then I'll have a look when i have time as to how to remove the collections stuff entirely and  only reference collections via the new parents code.